lib/board.rb

```diff
@@ -21,4 +21,8 @@ def next_token
   def is_available?(pos)
     @board[pos - 1].nil?
   end
+
+  def game_over?
+    @board.none? { |pos| pos.nil? }
```

Comment:
You might be able to write this as `@board.none?(&:nil?)` as well. No change needed, just wanted to throw that out there.

Reply:
I saw on the last PR you indicated that you changed all of the classes except for `Board` use attr_readers/attr_writers. Was there something about the `Board` that made this difficult here?

Reply:
I didn't want to put an attr_accessor and have the ability to modify board.  I guess I could put a private attr_accessor (somehow was getting some errors), but figured just same as having an instance variable.  I'll give a shot again.

Reply:
I guess you only need an attr_reader, and it works.  I made the change.

Reply:
👍 

In situations when you want a public reader and a private writer you can marke the  `attr_writer` as private.